src/components/Map.vue

```diff
@@ -174,19 +198,33 @@ export default {
 
       // TODO: "special" layers (ones handled by the map component itself, not GeoServer)
       _.each(this.layers, (layer) => {
-        if (layer.local !== true) {
```

Comment:
This is the part of the old code that treated local layers differently. We'll need to add something like this back in to fix the "All Fires, 2017" layer, I believe.